frontend/src/Pages/Student/Course/EnrolledCourses.jsx

```
@@ -144,21 +184,31 @@ export default function Courses() {
                                 >
-                                    Re-Enroll
-                                </button>
-                            )}
+                                    View Resources
+                                </a>
+
+                                {enrollment.canReEnroll && (
+                                    <button
+                                        onClick={() => handleReEnroll(enrollment.course_session.course.id)}
+                                        className="bg-blue-500 hover:bg-blue-600 text-white font-bold py-2 px-4 rounded-lg shadow-md"
+                                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)